### ISSUE TITLE
IOS API error: <_UIKBCompatInputView

### ISSUE THREAD

**Issue by ghq2340** [ionic-team/capacitor#835]
IOS reports this error when accessing an input type field on a **non-default loaded component**:

**API error: <_UIKBCompatInputView: 0x7f9c8dd1f860; frame = (0 0; 0 0); layer = <CALayer: 0x604000420f60>> returned 0 width, assuming UIViewNoIntrinsicMetric**

How to recreate this issue:

1. Create a new ionic (4) angular type app
ionic start myApp --type=angular (**select tabs template**, don't integrate with Cordova )

2. Replace the `<ion-content>` element with this code in myApp/src/app/about/about.page.html

`<ion-content padding>
  <ion-item>
    <ion-label color="primary" stacked>Stacked Label</ion-label>
    <ion-input type="email" placeholder="Email Input"></ion-input>
  </ion-item>
</ion-content>`

3. Install Capacitor ( https://capacitor.ionicframework.com/docs/getting-started/with-ionic )

4. Add ios:
    npx cap add ios

5. Build and launch IOS app:
     npm run build
     npx cap copy 
     npx cap open ios

5. Run the app then navigate to the "about" tab.

6. Click the input field

7. Check xcode's console:
API error: <_UIKBCompatInputView: 0x7fcba8519a50; frame = (0 0; 0 0); layer = <CALayer: 0x604000033600>> returned 0 width, assuming UIViewNoIntrinsicMetric

**Comment by jcesarmobile** [comment]
This is really thrown by the keyboard when it appears and not related to Capacitor itself, sometimes even native views have problems with auto layout and show this kind of messages, but if it doesn't affect the behaviour of the app you shouldn't worry, and it if affects the app you should report it to Apple.

**Comment by aiglesiasn** [comment]
Hi, Just to give some info. If you have that error on Xcode, if you're using the SIMULATOR you may try this:
- Hardware -> Keyboard -> Toggle Software Keyboard

Hope this helps.

**Comment by peterpeterparker** [comment]
For what's worth, I face that error with any Ionic input fields + Capacitor with the simulator too but only with iPhone 11 **pro** versions.

iPad, iPhone 8 or 11 all good, but with iPhone 11 Pro and iPhone 11 Pro Max I face this error.

**Comment by peterpeterparker** [comment]
Ok I've upgraded Xcode from version 11.1 to the last one 11.3. Even if I still do see in the console the above error at least now the keyboard and app aren't blocked anymore. Gui and interaction seems all ok aka works fine.

**Comment by asterism612** [comment]
how to solve this?

**Comment by juliannehalversen** [comment]
Running into the same issue with Cordova and the native ionic http plugin. Any solution?

**Comment by easyadin** [comment]
Hey guys, has anyone found a solution?

**Comment by puopg** [comment]
I want to just add a little something I experienced here as I feel it was a tad related.

I was seeing this error when focusing into an ion-input. My basic setup was like this:

```
<IonApp>
  <IonModal>
    <IonContent>
      <IonSlides>
        <IonSlide>
           <IonInput /> // This is what I was experiencing!
        </IonSlide>
      </IonSlides>
    </IonContent>
  </IonModal>
</IonApp>
```
So when developing, i would focus into the IonInput with no problems. I then added some more content into the view and so what was weird was suddenly when I focused into the IonInput, it took like 1 second before the cursor appeared. This was happening on browser, and on device. And removing the content (it was just a div with height), the delayed focus went away. 

Pertaining to this github issue, I was also seeing this `_UIKBCompatInputView` error. (Both with or without this additional content div)

So im stumped, but I noticed, when i focused into the input, the screen would shift by 1px up. Finally I realized what was going on was something was scrolling the IonContent. My guess is that was where the issue was spawning from. 

My take away from this was: Don't use an IonContent inside of an IonModal. Since the moment I removed that, no more scrolling (since the scrollable region went away) and no more delayed cursor. And no more _UIKBCompatInputView error!

Anyways, my TLDR is, Don't use IonContent inside of an IonModal. If it is supposed to be ok to do that, then I would like to investigate why the scrolling to the focus would cause such a major delay.
